hack/lib/etcd.sh

```diff
@@ -59,7 +59,7 @@ kube::etcd::validate() {
    version=$(etcd --version | head -n 1 | cut -d " " -f 3)
    if [[ $(kube::etcd::version "${ETCD_VERSION}") -gt $(kube::etcd::version "${version}") ]]; then
     kube::log::usage "etcd version ${ETCD_VERSION} or greater required."
-    kube::log::info "You can use 'hack/install-etcd.sh' to install a copy in third_party/."
+    kube::log::info "You can use 'hack/install-etcd.sh' to install a copy in third_party/. If old versions have been installed, please remove them first."
```

Comment:
I don't think older version needs to be removed. It should be ok as long as older version ETCD is not in PATH.

Reply:
In arktos.sh, it will check if the etcd is existed or not. If not, it will continue to install. We might change the following codes.

# install etcd if necessary
if ! [[ $(which etcd) ]]; then
  if ! [ -f "${KUBE_ROOT}/third_party/etcd/etcd" ]; then
     echo "cannot find etcd locally. will install one."
     ${KUBE_ROOT}/hack/install-etcd.sh
  fi

  export PATH=$PATH:${KUBE_ROOT}/third_party/etcd
fi

Reply:
Actually, arktos does not check the entire disk to see whether etcd is installed. As long as ETCD is not in the search path, it won't find it. This is how ETCD works. You can have two ETCD versions, updating PATH can start either one.  You don't need to remove (aka uninstall) other version of ETCD.

Reply:
OK. I will add comment to it in case that others might get confused.